### PR TITLE
[Disabled Tests] test/Error/rlexe.xml: tests tagged "fail" should be fixed and re-enabled #3118

### LINKED ISSUE #3118: [Disabled Tests] test/Error/rlexe.xml: tests tagged "fail" should be fixed and re-enabled
It appears that these tests are intended to fail, but the "fail" tag causes them to be excluded from all test runs.

They should be updated (probably to use the UnitTestFramework and/or baseline files) to detect that they fail as expected, and the "fail" tag should be removed so that these tests run.

```
test/Error/rlexe.xml-62-    <default>
test/Error/rlexe.xml-63-      <files>sourceInfo_00.js</files>
test/Error/rlexe.xml-64-      <baseline>sourceInfo_00.baseline</baseline>
test/Error/rlexe.xml:65:      <tags>fail</tags>
test/Error/rlexe.xml-66-    </default>
test/Error/rlexe.xml-67-  </test>
test/Error/rlexe.xml-68-  <test>
test/Error/rlexe.xml-69-    <default>
test/Error/rlexe.xml-70-      <files>sourceInfo_01.js</files>
test/Error/rlexe.xml-71-      <baseline>sourceInfo_01.baseline</baseline>
test/Error/rlexe.xml:72:      <tags>fail</tags>
test/Error/rlexe.xml-73-    </default>
test/Error/rlexe.xml-74-  </test>
test/Error/rlexe.xml-75-  <test>
test/Error/rlexe.xml-76-    <default>
test/Error/rlexe.xml-77-      <files>sourceInfo_10.js</files>
test/Error/rlexe.xml-78-      <baseline>sourceInfo_10.baseline</baseline>
test/Error/rlexe.xml:79:      <tags>fail</tags>
test/Error/rlexe.xml-80-    </default>
test/Error/rlexe.xml-81-  </test>
test/Error/rlexe.xml-82-  <test>
test/Error/rlexe.xml-83-    <default>
test/Error/rlexe.xml-84-      <files>sourceInfo_11.js</files>
test/Error/rlexe.xml-85-      <baseline>sourceInfo_11.baseline</baseline>
test/Error/rlexe.xml:86:      <tags>fail</tags>
test/Error/rlexe.xml-87-    </default>
test/Error/rlexe.xml-88-  </test>
test/Error/rlexe.xml-89-  <test>
test/Error/rlexe.xml-90-    <default>
test/Error/rlexe.xml-91-      <files>sourceInfo_12.js</files>
test/Error/rlexe.xml-92-      <baseline>sourceInfo_12.baseline</baseline>
test/Error/rlexe.xml:93:      <tags>fail</tags>
test/Error/rlexe.xml-94-    </default>
test/Error/rlexe.xml-95-  </test>
test/Error/rlexe.xml-96-  <test>
test/Error/rlexe.xml-97-    <default>
test/Error/rlexe.xml-98-      <files>sourceInfo_13.js</files>
test/Error/rlexe.xml-99-      <baseline>sourceInfo_13.baseline</baseline>
test/Error/rlexe.xml:100:      <tags>fail</tags>
test/Error/rlexe.xml-101-    </default>
test/Error/rlexe.xml-102-  </test>
test/Error/rlexe.xml-103-  <test>
test/Error/rlexe.xml-104-    <default>
test/Error/rlexe.xml-105-      <files>sourceInfo_20.js</files>
test/Error/rlexe.xml-106-      <baseline>sourceInfo_20.baseline</baseline>
test/Error/rlexe.xml:107:      <tags>fail</tags>
test/Error/rlexe.xml-108-    </default>
test/Error/rlexe.xml-109-  </test>
test/Error/rlexe.xml-110-  <test>
```

### PR DESCRIPTION
**Pull request by tcare**
Re-enabled and updated prehistoric error source info baselines.

Stack information is such a luxury. :)

Fixes #3118

### CODE REVIEW

**Comment by akroshg** [comment]
Would it be good idea to use unittestframework to assert for the error?

**Comment by tcare** [comment]
These tests wanted to test source offsets which UnitTestFramework won't cover.

**Comment by dilijev** [comment]
Maybe use https://github.com/Microsoft/ChakraCore/blob/master/test/UnitTestFramework/TrimStackTracePath.js to normalize the stack traces?

**Comment by tcare** [comment]
Thanks for Doug for his input on what to do here.

I added the flag to make the errors more jshost-like, but throwing non-objects gives a different error in ch. In this case, it outputs the full path to the filename. I changed ch to just output the filename in this case, bringing it into line with similar situations in the surrounding code.

I also decided to have different baselines for Full and make it work like DebuggerCommon. Please see the corresponding ChakraFull review.

**Comment by tcare** [comment]
In addition, I tried to make the non-object error look like other error messages, but context info like line numbers was not available. It seems we need to make more substantial changes to ch to bring this in line with jshost, but that is out of the scope of this change.

File: bin/ch/WScriptJsrt.cpp [hunk 1]
```diff
@@ -1369,8 +1369,14 @@ bool WScriptJsrt::PrintException(LPCSTR fileName, JsErrorCode jsErrorCode)
                 if (errorCode != JsErrorCode::JsNoError || propertyType == JsUndefined)
                 {
                     const char *fName = fileName != nullptr ? fileName : "(unknown)";
+
+
```
Comment (dilijev) [inline bin/ch/WScriptJsrt.cpp]: nit: extra blank line

**Pull request merged**